src/init/entry.js

```diff
@@ -3,31 +3,22 @@
 import React from 'react';
 import { render } from 'react-dom';
 import log from 'loglevel';
-import i18n from '@dhis2/d2-i18n';
 import 'typeface-roboto';
```

Comment:
since we are touching this file I have a suggestion. 

Could we move this line and the line about `import './addRxjsOperators';` to [here](https://github.com/dhis2/capture-app/blob/refactor-metadata-fetching/src/index.js)?

Looks like this is a more applicable place for these kind of imports

Reply:
Pretty sure at one point it was imported in the file you're mentioning. So I'm a bit wary about moving it without properly testing it.

Reply:
Okay good point. Thats out of scope as well. Maybe I could open a PR at some point with all those suggestions.